core/ecschema-editing/package.json

```diff
@@ -36,6 +36,7 @@
     "@bentley/bentleyjs-core": "workspace:*",
     "@bentley/build-tools": "workspace:*",
     "@bentley/eslint-plugin": "workspace:*",
+    "@bentley/imodeljs-common": "workspace:*",
     "@bentley/imodeljs-i18n": "workspace:*",
```

Comment:
Can we drop the i18n as a dependency? If needed, lets move it to a devDep if it's used within tests.

Reply:
It is needed by tests; moved to devDep.